### PR TITLE
[docs] Conda update

### PR DESCRIPTION
**Pull request by artanokhov**
### Details:
 - Added section with info regarding compiling with OpenVINO from Conda-Forge
 - Added conda-forge installation to MacOS install header

### CODE REVIEW

File: docs/install_guides/installing-openvino-conda.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 .. note::
 
-   Installing OpenVINO Runtime from Conda Forge is recommended for C++ developers, as it provides only the C++ Runtime API.
+   Installing OpenVINO™ Runtime from Conda Forge is recommended for C++ developers, as it provides only the C++ Runtime API.
```
Comment (tsavina) [inline docs/install_guides/installing-openvino-conda.md:7]:
```suggestion
   Installing OpenVINO Runtime from Conda Forge is recommended for C++ developers, as it provides only the C++ Runtime API.
```

File: docs/install_guides/installing-openvino-conda.md [hunk 2]
```diff
@@ -25,7 +25,7 @@
    `Anaconda distribution page <https://www.anaconda.com/products/individual/>`__
 
 
-Installing OpenVINO Runtime with Anaconda Package Manager
+Installing OpenVINO™ Runtime with Anaconda Package Manager
```
Comment (tsavina) [inline docs/install_guides/installing-openvino-conda.md:28]:
```suggestion
Installing OpenVINO Runtime with Anaconda Package Manager
```

File: docs/install_guides/installing-openvino-conda.md [hunk 3]
```diff
@@ -44,14 +44,35 @@ Installing OpenVINO Runtime with Anaconda Package Manager
 
       conda update --all
 
-3. Install the OpenVINO Runtime package:
+3. Install the OpenVINO™ Runtime package:
```
Comment (tsavina) [inline docs/install_guides/installing-openvino-conda.md:47]:
```suggestion
3. Install the OpenVINO Runtime package:
```

File: docs/install_guides/installing-openvino-macos-header.md [hunk 4]
```diff
@@ -9,12 +9,14 @@
    From Archive <openvino_docs_install_guides_installing_openvino_from_archive_macos>
    From PyPI <openvino_docs_install_guides_installing_openvino_pip>
    Using Homebrew <openvino_docs_install_guides_installing_openvino_brew>
+   Using Conda Forge <openvino_docs_install_guides_installing_openvino_conda>
```
Comment (tsavina) [inline docs/install_guides/installing-openvino-macos-header.md:12]: minor comment - for Linux conda install goes before homebrew page

File: docs/install_guides/installing-openvino-conda.md [hunk 5]
```diff
@@ -44,14 +44,35 @@ Installing OpenVINO Runtime with Anaconda Package Manager
...
 
    .. code-block:: sh
 
       conda install -c conda-forge openvino=2023.0.0
 
    Congratulations! You have finished installing OpenVINO Runtime.
 
+Compiling with OpenVINO™ Runtime from Conda-Forge
```
Comment (tsavina) [inline docs/install_guides/installing-openvino-conda.md:55]:
```suggestion
Compiling with OpenVINO Runtime from Conda-Forge
```

File: docs/install_guides/installing-openvino-conda.md [hunk 6]
```diff
@@ -44,14 +44,35 @@ Installing OpenVINO Runtime with Anaconda Package Manager
...
+###########################################################
+
+When linking OpenVINO™ libraries from Conda, ensure you have the necessary Conda compilers installed.
```
Comment (tsavina) [inline docs/install_guides/installing-openvino-conda.md:58]:
```suggestion
When linking OpenVINO libraries from Conda, ensure that you have the necessary Conda compilers installed. 
```

File: docs/install_guides/installing-openvino-conda.md [hunk 7]
```diff
@@ -44,14 +44,35 @@ Installing OpenVINO Runtime with Anaconda Package Manager
...
+###########################################################
+
+When linking OpenVINO™ libraries from Conda, ensure you have the necessary Conda compilers installed. 
+To do so, run the following command in your Conda environment:
+
+.. code-block:: sh
+
+    conda install cmake c-compiler cxx-compiler make
+
+It is crucial to reactivate your Conda environment after installation of the compilers.
```
Comment (tsavina) [inline docs/install_guides/installing-openvino-conda.md:65]:
```suggestion
It is crucial to reactivate your Conda environment after installing the compilers. 
```

File: docs/install_guides/installing-openvino-conda.md [hunk 8]
```diff
@@ -44,14 +44,35 @@ Installing OpenVINO Runtime with Anaconda Package Manager
...
 
+Compiling with OpenVINO™ Runtime from Conda-Forge
+###########################################################
+
+When linking OpenVINO™ libraries from Conda, ensure you have the necessary Conda compilers installed. 
+To do so, run the following command in your Conda environment:
+
+.. code-block:: sh
+
+    conda install cmake c-compiler cxx-compiler make
```
Comment (ilya-lavrenov) [inline docs/install_guides/installing-openvino-conda.md:63]: it's turned out the compilers are not required on macOS (at least 13.x). But maybe it depends on the macOS version and on 11.x with older libc++ we still require conda compilers.

Could you please try macOS 11? and Windows ?

File: docs/install_guides/installing-openvino-conda.md [hunk 9]
```diff
@@ -44,14 +44,35 @@ Installing OpenVINO Runtime with Anaconda Package Manager
...
+This step ensures that all the environment variables are set correctly for successful linkage.
+
+To reactivate your Conda environment, execute the following command:
+
+.. code-block:: sh
+    conda deactivate
+    conda activate <your_environment_name>
+
+Replace <your_environment_name> with the name of your Conda environment. 
+Once reactivated, you can proceed with linking the OpenVINO™ libraries, making sure all the necessary environment variables are properly set.
```
Comment (tsavina) [inline docs/install_guides/installing-openvino-conda.md:75]:
```suggestion
Once you have reactivated your Conda environment, make sure that all the necessary environment variables are properly set and proceed with linking the OpenVINO libraries.
```

File: docs/install_guides/installing-openvino-conda.md [hunk 10]
```diff
@@ -44,14 +44,35 @@ Installing OpenVINO Runtime with Anaconda Package Manager
...
+
+    conda install cmake c-compiler cxx-compiler make
+
+It is crucial to reactivate your Conda environment after installation of the compilers. 
+This step ensures that all the environment variables are set correctly for successful linkage.
+
+To reactivate your Conda environment, execute the following command:
+
+.. code-block:: sh
+    conda deactivate
```
Comment (tsavina) [inline docs/install_guides/installing-openvino-conda.md:71]:
```suggestion
.. code-block:: sh

    conda deactivate
```

**Review by yury-gorbachev — APPROVED** [review]

File: docs/install_guides/installing-openvino-conda.md (hunk 8 above)
Comment (artanokhov) [reply docs/install_guides/installing-openvino-conda.md:63]: It's working on Windows and MacOS without installing compilers from Conda

Comment (artanokhov) [reply docs/install_guides/installing-openvino-conda.md:63]: I've updated title

**Review by ilya-lavrenov — APPROVED** [review]

File: docs/install_guides/installing-openvino-conda.md [hunk 11]
```diff
@@ -52,6 +52,28 @@ Installing OpenVINO Runtime with Anaconda Package Manager
...
+    conda install cmake c-compiler cxx-compiler make
+
+It is crucial to reactivate your Conda environment after installing the compilers. 
+This step ensures that all the environment variables are set correctly for successful linkage.
+
+To reactivate your Conda environment, execute the following command:
+
+.. code-block:: sh
+
+    conda deactivate
```
Comment (ilya-lavrenov) [inline docs/install_guides/installing-openvino-conda.md:72]: it's turned out that to re-activate, we can just use `conda activate <env>`

**Pull request merged**